docs/workflows-setup.md

```diff
@@ -50,3 +50,24 @@ be **Test all targets**).
 
 After you save your changes, pull requests will not be mergeable unless
 the tests pass on BuildBuddy.
+
+## Building in the workflow runner environment
+
+BuildBuddy workflows execute using Docker on an Ubuntu 18.04 base image
```

Comment:
Can you change this to something like:

"BuildBuddy workflows execute using Docker on a recent Ubuntu (18.04 at the time of this writing) base image"

I don't want us locked into a particular version by default.

Reply:
Done